src/config_handling.py

```diff
@@ -36,7 +39,7 @@ def init_config(app):
 
     for config in all_configs:
         if config in file_configs:
-            value = get_config_from(app, config + "_FILE")
+            value = get_config_from(app, config)
         else:
             value = os.environ.get(config, get_default_for(config))
```

Comment:
I think, you can simplify these lines just to

```suggestion
        value = get_config_from(app, config)
``` 

because you already check if `config + "_FILE"` is present and read alternatively from `os.environ`.